pages/v3/Working-With-Animations.md

```diff
@@ -12,30 +12,37 @@ Very few games are made that don't need animations. Duality supports animations
 So, what is needed for an animation?
 A spritesheet. A spritesheet is nothing more than a sequence of images that when shown in sequence form an animation.
 So, let's just use this one (Downloaded from OpenGameArt, character created by Bevouliin):
-![](../img/Animation/Animation1.png)
 
-Save it somewhere on your computer...
+![](../img/Animation/Animation1.png)
 
-And open Duality.
+Save it somewhere on your computer and open Duality.
 
 Drag and drop the spritesheet to the project view.
 
 Looking to the right you see a few fields that relate to animation. In our example, you see Billy six times doing different things.
+
 ![](../img/Animation/Animation2.png)
-Insert 3 in the AnimCols and 2 in the AnimRows. AnimFrameBorder is only if you have borders between the frames.
+
+Insert 3 in the SpriteSheetCoumns and 2 in the SpriteSheetRows. SpriteFrameBorder is only if you have borders between the frames and click the Import button.  This will create your Atlas for you.  It should now look like this:
+
+![](../img/Animation/Animation2_5.png)
 
 Create a new GameObject by right-clicking in the scene view and give it a name: Billy (I call it that)
-In Billy, add a new AnimSpriteRenderer Component.
+In Billy, add a new SpriteAnimator (New -> Graphics -> SpriteAnimator).  This will automatically create a Transform and SpriteRenderer in your GameObject.
 
 ![](../img/Animation/Animation3.png)
 
-You will Duality's logo appearing. That's ok. Drag Billy's Pixmap over AnimSpriteRenderer and you'll see Billy in your scene.
+You will Duality's logo appearing. That's ok. In the SpriteRenderer, drag Billy's Pixmap from the Project View window over the SharedMaterial and update the width and height of the SpriteRenderer to match the size of the spritesheet to a height of 205 and a width of 231.  Set the AnimDuration to 1.0, the FirstFrame to 0 and the FrameCount to 4 (The walk animation is 4 frames long).
```

Comment:
typo: You will _see_ Duality...

in addition, there's something wrong setting the SpriteRenderer's size to 231x205; that's the spritesheet's original size, but it contains a non-square grid of 3x2 sprites, which means that the ratio is off and it is even scaled up. The "correct" values to have the renderer display a 1:1 version of the single sprite should be 

| -35.0 | -51.25 |
| ---:| ---:|
| 77.0 | 102.5 |

(size / grid, then /2 to have the single sprite's offset)

Reply:
For the scaled up part, maybe it's enough to just add a remark about it

Reply:
Thanks for the feedback - updating shortly.